{"version":"3.8.0","messages":["\nAn error occurred while loading ./spec/secure_password_spec.rb.\n\u001b[31mFailure/Error: \u001b[0mrequire_relative \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31m../config/environment.rb\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mGem::LoadError:\u001b[0m\n\u001b[31m  Specified 'sqlite3' for database adapter, but the gem is not loaded. Add `gem 'sqlite3'` to your Gemfile (and ensure its version is at the minimum required by ActiveRecord).\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activerecord-5.1.0/lib/active_record/connection_adapters/connection_specification.rb:188:in `rescue in spec'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activerecord-5.1.0/lib/active_record/connection_adapters/connection_specification.rb:185:in `spec'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activerecord-5.1.0/lib/active_record/connection_adapters/abstract/connection_pool.rb:880:in `establish_connection'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activerecord-5.1.0/lib/active_record/connection_handling.rb:58:in `establish_connection'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/sinatra-activerecord-2.0.13/lib/sinatra/activerecord.rb:48:in `database='\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/sinatra-2.0.5/lib/sinatra/base.rb:1233:in `set'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/sinatra-2.0.5/lib/sinatra/base.rb:1925:in `block (2 levels) in delegate'\u001b[0m\n\u001b[36m# ./config/environment.rb:5:in `block in \u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/sinatra-2.0.5/lib/sinatra/base.rb:1426:in `configure'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/sinatra-2.0.5/lib/sinatra/base.rb:1925:in `block (2 levels) in delegate'\u001b[0m\n\u001b[36m# ./config/environment.rb:4:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# ./spec/spec_helper.rb:1:in `require_relative'\u001b[0m\n\u001b[36m# ./spec/spec_helper.rb:1:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-5.1.0/lib/active_support/dependencies.rb:292:in `require'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-5.1.0/lib/active_support/dependencies.rb:292:in `block in require'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-5.1.0/lib/active_support/dependencies.rb:258:in `load_dependency'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-5.1.0/lib/active_support/dependencies.rb:292:in `require'\u001b[0m\n\u001b[36m# ./spec/secure_password_spec.rb:1:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-5.1.0/lib/active_support/dependencies.rb:286:in `load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-5.1.0/lib/active_support/dependencies.rb:286:in `block in load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-5.1.0/lib/active_support/dependencies.rb:258:in `load_dependency'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-5.1.0/lib/active_support/dependencies.rb:286:in `load'\u001b[0m\n\u001b[36m# ------------------\u001b[0m\n\u001b[36m# --- Caused by: ---\u001b[0m\n\u001b[36m# Gem::LoadError:\u001b[0m\n\u001b[36m#   can't activate sqlite3 (~\u003e 1.3.6), already activated sqlite3-1.4.0. Make sure all dependencies are added to Gemfile.\u001b[0m\n\u001b[36m#   /usr/local/rvm/gems/ruby-2.6.1/gems/activerecord-5.1.0/lib/active_record/connection_adapters/sqlite3_adapter.rb:10:in `\u003ctop (required)\u003e'\u001b[0m\n","No examples found."],"examples":[],"summary":{"duration":0.000384715,"example_count":0,"failure_count":0,"pending_count":0,"errors_outside_of_examples_count":2},"summary_line":"0 examples, 0 failures, 2 errors occurred outside of examples"}